Don't build bundled gumbo-parser. We'll use MacPorts gumbo-parser.
https://github.com/litehtml/litehtml/issues/151
--- CMakeLists.txt.orig	2018-09-16 14:14:44.000000000 -0500
+++ CMakeLists.txt	2021-04-14 15:53:14.000000000 -0500
@@ -2,7 +2,6 @@
 
 project(litehtml CXX)
 
-add_subdirectory(src/gumbo)
 
 set(SOURCE_LITEHTML
     src/background.cpp
--- src/document.cpp.orig	2018-09-16 14:14:44.000000000 -0500
+++ src/document.cpp	2021-04-14 16:41:48.000000000 -0500
@@ -24,7 +24,6 @@
 #include <math.h>
 #include <stdio.h>
 #include <algorithm>
-#include "gumbo/gumbo.h"
 #include "utf8_strings.h"
 
 litehtml::document::document(litehtml::document_container* objContainer, litehtml::context* ctx)
--- src/document.h.orig	2018-09-16 14:14:44.000000000 -0500
+++ src/document.h	2021-04-14 16:41:51.000000000 -0500
@@ -4,7 +4,7 @@
 #include "style.h"
 #include "types.h"
 #include "context.h"
-#include "gumbo/gumbo.h"
+#include <gumbo.h>
 
 namespace litehtml
 {
